x-pack/plugins/ml/public/application/timeseriesexplorer/components/forecasting_modal/forecasting_modal.js

```diff
@@ -162,8 +170,8 @@ export class ForecastingModal extends Component {
       jobOpeningState: PROGRESS_STATES.WAITING,
     });
 
-    mlJobService
-      .openJob(this.props.job.job_id)
+    this.context.services.mlServices.mlApiServices
+      .openJob({ jobId: this.props.job.job_id })
```

Comment:
For a closed job, I'm finding that the forecast fails to re-open the job:

<img width="936" alt="Screenshot 2024-07-22 at 17 55 06" src="https://github.com/user-attachments/assets/458018e4-e4c9-4e5c-898d-7f9fe155fbd4">

Reply:
Fixed in bd7d459c72ebf7c5f80a13fa24011b269ad32e82